src/tendencies/vertical_diffusion_boundary_layer.jl

```diff
@@ -263,6 +265,7 @@ function get_surface_fluxes!(Y, p, t, colidx, ::VerticalDiffusion)
         uₕ_int_phys,
         params,
         coupling,
+        buoy_flux_surface,
```

Comment:
Shouldn't we just use `sfc_conditions[colidx].buoy_flux` in place of where this is used instead?

Reply:
Or are they allowed to have different values for some reason?

Reply:
I think you are right. For some reason I thought we are not caching the full `sfc_conditions`. Thanks. I'll open a PR with a fix.

Reply:
And really thank you for reviewing!